doc/Aliases-and-defaults.md

```diff
@@ -0,0 +1,41 @@
+# Aliases and Defaults
+
+All this can amount to a lot of typing, so BiblioPixel gives you some shortcuts.
+
+## Typename expansion.
+
+If you are only specifying a `typename` for a component, then you don't have to specify the entire dictionary, just the name of the class.
```

Comment:
This line is super-long!

Reply:
This whole file is just copied direct from the wiki. It's there as an example. Also... YOU wrote it!